docs/development.md

```diff
@@ -90,12 +90,24 @@ Terminology we use:
 
 ### Reviewing a pull request
 
-What to look out for when reviewing a pull request:
+#### What to look out for when reviewing a pull request:
 
   - Have tests been added?
   - Does this feature / fix work locally for me? 
   - Am I able to understand the code correctly / have comments been added to the code?
 
+#### Prow best practices
+
+  - Get used to how prow works and its commands. 
+    - Prow uses [OWNERS](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md) files to determine who can approve and lgtm a PR.
+    - List of possible bot [commands](https://prow.k8s.io/command-help)
```

Comment:
Link to https://deck-ci.svc.ci.openshift.org/command-help, that is the wrong Prow

Reply:
Thank you for checking.
fixing in https://github.com/redhat-developer/odo/pull/1438